// warnerburg-common/common.properties.js
var commonData = {
    mainCssUrl:     '/includes/css/warnerburg.css',
    home:         {
        url:        '/',
        image:      '/images/navigation/homedarks.png'
    },
    gallery:         {
        url:        '/gallery',
        image:      '/images/navigation/gallerydarks.png'
    },
    'gallery-color':         {
        url:        '/gallery/color'
    },
    'gallery-pencil':         {
        url:        '/gallery/pencil'
    },
    'gallery-progression':         {
        url:        '/gallery/progression'
    },
    store:         {
        url:        '/store',
        image:      '/images/navigation/storedarks.png'
    },
    comic:         {
        url:        '/comic',
        image:      '/images/navigation/comicdarks.png'
    },
    archives:         {
        url:        '/comic/archives'
    },
    info:         {
        url:        '/info',
        image:      '/images/navigation/infodarks.png'
    },
    links:      {
        url:        '/links',
        image:      '/images/navigation/linksdark.jpg'
    },
    authorPicServerFolder: '/app/client/images/details/authors/',
    comicImageServerFolder: '/app/client/images/comic/',
    authorPicClientFolder: '/images/details/authors/',
    comicImageClientFolder: '/images/comic/',
    allowedImageExtensions: ['.jpg', '.png'],
    publishDateTimeAdminFormat: {
        angular: 'MM/dd/yyyy HH:mm',
        display: 'mm/dd/yy'
    }
};

// this file's getting included client-side as a js include and server-side as a node module;
// the below code makes it so the module-specific stuff only happens server-side.
if (typeof module !== 'undefined' && module.exports) {
    module.exports = commonData;
}
