// warnerburg-common/common.properties.js
var commonData = {
    mainCssUrl:     '/includes/css/warnerburg.css',
    home:         {
        url:        '/',
        image:      '/images/navigation/homedarks.png'
    },
    gallery:         {
        url:        '/gallery',
        image:      '/images/navigation/gallerydarks.png'
    },
    'gallery-color':         {
        url:        '/gallery/color',
        serverFolder: '/app/client/images/gallery/color/main/',
        clientFolder: '/images/gallery/color/main/'
    },
    'gallery-pencil':         {
        url:        '/gallery/pencil',
        serverFolder: '/app/client/images/gallery/pencil/',
        clientFolder: '/images/gallery/pencil/'
    },
    'gallery-progression':         {
        url:        '/gallery/progression',
        serverFolder: '/app/client/images/gallery/progression/',
        clientFolder: '/images/gallery/progression/'
    },
    'author-pic':         {
        serverFolder: '/app/client/images/details/authors/',
        clientFolder: '/images/details/authors/'
    },
    'archives-thumbnail':         {
        serverFolder: '/app/client/images/archive/thumbnail/',
        clientFolder: '/images/archive/thumbnail/'
    },
    'archives-description-image':         {
        serverFolder: '/app/client/images/archive/description/',
        clientFolder: '/images/archive/description/'
    },
    store:         {
        url:        '/store',
        image:      '/images/navigation/storedarks.png'
    },
    comic:         {
        url:        '/comic',
        image:      '/images/navigation/comicdarks.png',
        serverFolder: '/app/client/images/comic/',
        clientFolder: '/images/comic/'
    },
    archives:         {
        url:        '/comic/archives'
    },
    info:         {
        url:        '/info',
        image:      '/images/navigation/infodarks.png'
    },
    links:      {
        url:        '/links',
        image:      '/images/navigation/linksdark.jpg'
    },
    allowedImageExtensions: ['.jpg', '.png'],
    publishDateTimeAdminFormat: {
        angular: 'MM/dd/yyyy HH:mm',
        display: 'mm/dd/yy'
    }
};

// this file's getting included client-side as a js include and server-side as a node module;
// the below code makes it so the module-specific stuff only happens server-side.
if (typeof module !== 'undefined' && module.exports) {
    module.exports = commonData;
}
